configuration.rst

```diff
@@ -485,48 +485,45 @@ You can reference environment variables using the special syntax
 ``%env(ENV_VAR_NAME)%``. The values of these options are resolved at runtime
 (only once per request, to not impact performance).
 
-This example shows how you could configure the database connection using an env var:
+This example shows how you could configure the application secret using an env var:
 
 .. configuration-block::
 
     .. code-block:: yaml
 
-        # config/packages/doctrine.yaml
-        doctrine:
-            dbal:
+        # config/packages/framework.yaml
+        framework:
+            secret: '%env(string:SECRET)%'
                 # by convention the env var names are always uppercase
-                url: '%env(resolve:DATABASE_URL)%'
+                secret: '%env(APP_SECRET)%'
             # ...
 
     .. code-block:: xml
 
-        <!-- config/packages/doctrine.xml -->
+        <!-- config/packages/framework.xml -->
         <?xml version="1.0" encoding="UTF-8" ?>
         <container xmlns="http://symfony.com/schema/dic/services"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:doctrine="http://symfony.com/schema/dic/doctrine"
             xsi:schemaLocation="http://symfony.com/schema/dic/services
-                https://symfony.com/schema/dic/services/services-1.0.xsd
-                http://symfony.com/schema/dic/doctrine
```

Comment:
very unsure of those, may need careful review :s

Reply:
line 509 should be kept :)
Line 507 should be changed to:
`xmlns:framework="http://symfony.com/schema/dic/symfony`
Finally line 510 and 511 should be changed to refer to:
`http://symfony.com/schema/dic/symfony` and
`https://symfony.com/schema/dic/symfony/symfony-1.0.xsd`